Scripts/ci/build-sim.sh

```diff
@@ -1,9 +1,5 @@
 #!/bin/bash
 
-set -x
```

Comment:
this might still be useful for debugging

Reply:
Well, this script is mostly what kind of files were in the zip, so the -x flag didn't help so much. Only the normal zip command output helped to debug